qdev_wrappers/alazar_controllers/acquisition_parameters.py

```diff
@@ -21,9 +21,8 @@ class AcqVariablesParam(Parameter):
 
     def __init__(self, name, instrument, check_and_update_fn,
                  default_fn=None, initial_value=None):
-        super().__init__(name)
+        super().__init__(name, initial_cache_value=initial_value)
```

Comment:
it might be that initial_cache_value was introduced in version later than 0.9.0

Reply:
There is no specific mentioning of initial_cache_value in qcodes change logs. But, I found this:
initial_cache_value – Value to set the cache of the parameter to at the end of its initialization (this is equivalent to calling parameter.cache.set(initial_cache_value) after parameter initialization).
So I think having qcodes 0.9.0 as the minimum qcodes version should not be an issue regarding this initial_cache_value.

Reply:
@FarBo Have a look here https://qcodes.github.io/Qcodes/changes/0.15.0.html#improved :)

Reply:
> There is no specific mentioning of initial_cache_value in qcodes change logs

unfortunately it's mentioned as initial_cache_val in 0.15.0 changelog https://github.com/QCoDeS/Qcodes/releases/tag/v0.15.0

@FarBo 

Another way to figure out when the change was introduces is to use "blame" (in github or in git itself) on the parameters.py file, and then the cool feature of github that shows tags that include a given commit, which results in this:

![image](https://user-images.githubusercontent.com/15662810/91297541-cc7fd480-e79e-11ea-99d0-545ae6cbb954.png)

So initial_cache_value is there from qcodes 0.15.0. Hence if it is to be used, then the minimal version needs to be bumped to 0.15.0.

Reply:
I see. It is mentioned as initial_cache_val in the change log, and that was the reason I couldn't find it. Blame method is very cool, though. thanks Mikhail for the point.
I bump the version to 0.15.0 and make a new commit.